### ISSUE TITLE
hint loss 不是计算logits的L2距离吧

### ISSUE THREAD

**Issue by forseeking** [SsisyphusTao/SSD-Knowledge-Distillation#9]
https://github.com/SsisyphusTao/SSD-Knowledge-Distillation/blob/0597fbee635afcf0b8710ba3a9e40ab9f010aea5/nets/multibox_loss.py#L158

**Comment by SsisyphusTao** [comment]
> https://github.com/SsisyphusTao/SSD-Knowledge-Distillation/blob/0597fbee635afcf0b8710ba3a9e40ab9f010aea5/nets/multibox_loss.py#L158

根据论文原文意思确实不是，但论文也没有给出hint loss的具体实现，这个是我根据自己理解实验试出来的一个有效的方法，具体可以看(https://zhuanlan.zhihu.com/p/260370225)这篇里面有说明
